graphdatascience/tests/integrationV2/procedure_surface/cypher/conftest.py

```diff
@@ -2,27 +2,50 @@
 from pathlib import Path
 from typing import Generator
 
+import dotenv
```

Comment:
need to adjust our test requirements txt to reflect this.
for now dotenv was only used in examples